index.js

```diff
@@ -37,6 +37,8 @@ module.exports.filter = shouldCompress
 
 var cacheControlNoTransformRegExp = /(?:^|,)\s*?no-transform\s*?(?:,|$)/
 
+var encodingSupported = ['*', 'gzip', 'deflate', 'identity']
```

Comment:
Why are we only supporting these?

Reply:
These are the options supported by the package at the moment; support for Brotli is still in progress

Reply:
Okay, then I will try to get a better understanding of this and get back to you.